components/message/MessageService.cs

```diff
@@ -11,11 +11,11 @@ namespace AntDesign
     /// <summary>
     /// Message Service
     /// </summary>
-    public class MessageService
+    public class MessageService: IMessageService
     {
-        internal event Action<MessageGlobalConfig> OnConfig;
-        internal event Func<MessageConfig, Task> OnOpening;
-        internal event Action OnDestroy;
+        public event Action<MessageGlobalConfig> OnConfig;
+        public event Func<MessageConfig, Task> OnOpening;
+        public event Action OnDestroy;
```

Comment:
Please keep these internal.

Reply:
Okay. I removed those from the interface and made them internal again on the concrete class. The internal C# for the Message component will have to stay using the concrete class in that case.